catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonBaseIT.java

```diff
@@ -712,11 +727,11 @@ public void testAlterPaimonTable() {
     // update column position
     Column col1 = Column.of("name", Types.StringType.get(), "comment");
     Column col2 = Column.of("address", Types.StringType.get(), "comment");
-    Column col3 = Column.of("date_of_birth", Types.DateType.get(), "comment");
```

Comment:
it will throw an exception about `InvalidOperationException(message:The following columns have types incompatible with the existing columns in their respective positions` for hive backend, and it is an issue of hms not lakehouse. Therefor, i changed the column type here to ignore this problem.

as shown below, hudi and iceberg both have this problem.
https://github.com/apache/hudi/issues/2212
https://github.com/apache/iceberg/issues/1092

Reply:
got it